docs/index.rst

```diff
@@ -1,53 +1,211 @@
+           the `Livepatch documentation`_, or find out
+           :doc:`how to enable it<howtoguides/enable_livepatch>`.
+
+       * **Landscape** (coming soon!)
+
+         `Landscape`_
+
+       Want more information?
+
+    .. tab-item:: Certification and hardening
+
+       Ubuntu provides security compliance, `certifications, and hardening`_:
+
+       * **FIPS 140-2**
+
+         * `FIPS 140-2 Certified Modules`_ are available through ``fips``.
+
+         * Compliant but non-certified patches are available through
+           ``fips-updates``.
+
+         * For more details about managing FIPS, check out this guide. <-- note
+           we need a guide, which also includes which services are
+           incompatible with fips   
+
+       * **CIS benchmarking**
+
+         * Center for Internet Security `(CIS) Benchmark`_ tooling is available
+           with ``cis``, up to (and including) 20.04 LTS.
+         * In the 20.04 LTS we introduced the `Ubuntu Security Guide (USG)`_
+           tooling through ``usg``, which replaces ``cis`` in releases after
+           20.04 LTS. 
+
+         * Find out :doc:`how to manage CIS and USG<howtoguides/enable_cis>`
+           with the Pro Client.
+
+       * **Common Criteria**
+
+         * `Common Criteria EAL2 (CC EAL)`_ certification tooling is available
+           through ``cc-eal``. The Ubuntu 18.04 LTS and 16.04 LTS have both
+           been certified.
+
+         * Find out :doc:`how to enable CC EAL<howtoguides/enable_cc>` on
+           your LTS.
+
+       Want more information?
+
+    .. tab-item:: Personal user on LTS
+
+       * **Expanded Security Maintenance (ESM)**
+
+         If you're using Pro on an LTS machine (Desktop or Server) and don't
+         want to upgrade to a newer LTS yet, you can get additional security
+         support through `Expanded Security Maintenance (ESM)`_. We recommend
+         it for everyone using Pro, and it's enabled by default. It includes
+         two services:
+
+         * ``esm-infra`` provides security updates for packages in the Ubuntu
+           'main' repository (packages from Ubuntu) for five additional years
+           beyond the standard five years of security maintenance of 'main'
+           packages.
+
+         * ``esm-apps`` covers packages in the 'universe' repository (which
+           are from Debian and the community) for ten years after the LTS
+           release. 
+
+         * Read more
+           :doc:`about ESM-Apps and ESM-Infra<explanations/about_esm>`.
+
+       * **Livepatch**
+
+         Ubuntu `Livepatch`_ removes the need to immediately reboot your
+         machine after a patch is applied to the kernel, so you can perform
+         device reboots when it's convenient for you. Although it's
+         lightweight, it may not be suitable on a very small system with
+         limited memory. Check our `list of supported kernels`_ if you're
```

Comment:
We should check with livepatch team before asserting this. We do enable livepatch by default on all systems.

Reply:
Who would be the best person to speak to, do you think? I can reach out